.github/workflows/requirements.yml

```diff
@@ -0,0 +1,66 @@
+# cspell:ignore noreply
+
+name: Upgrade requirements
+
+on:
+  schedule:
+    - cron: "0 2 * * 1"
```

Comment:
We can think about running this every 2 days, since it wont actually do anything if there are no changes. That way we cover more of the package combinations that exist and can achieve a bit higher compatibility.

Reply:
Considering the large number of packages in `requirements-dev.txt`, I'm afraid this would result in an upgrade PR every two days. Just one of those indirect dependencies would have to release an upgrade and the cron job would create a new PR.

Reply:
Hmm, but I think this would help us. Rather have many small updates than one big one. So in case there is some compatibility issue, we can easier figure out which package is causing it.

Reply:
Ok can change it. I just thought it might clog the commit history, see e.g.
https://github.com/peterdemin/pip-compile-multi/commits/master

Reply:
I see. We can always reduce it, if it would be too annoying